pkg/controller/direct/gkehub/featuremembership_controller.go

```diff
@@ -121,6 +121,14 @@ func (a *gkeHubAdapter) Find(ctx context.Context) (bool, error) {
 
 // Delete implements the Adapter interface.
 func (a *gkeHubAdapter) Delete(ctx context.Context) (bool, error) {
+	exist, err := a.Find(ctx)
+	if err != nil {
+		return false, fmt.Errorf("finding feature for %s:%w", a.featureID, err)
```

Comment:
```suggestion
		return false, fmt.Errorf("finding feature membership for %s:%w", a.featureID, err)
```

maybe it makes more sense but if so please feel free to follow up in a separate PR

Reply:
Actually, the Find gets an Gkehubfeature resource back, there is no GCP resource called featuremembership